cron/cron.txt

```diff
@@ -10,4 +10,4 @@
 20      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/monthly.sh
 30      4       1       1       *       bash /home/ubuntu/code/CantusDB/cron/postgres/yearly.sh
 40      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_next_chant_fields; bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_is_last_chant_in_feast
-50      4       *       *       0       /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org -d www.cantusdatabase.org --renew-hook "nginx -s reload"
+50      4       7       *       *       /usr/local/bin/docker-compose run nginx lego -d cantusdatabase.org -d www.cantusdatabase.org --http --http.webroot /var/www/lego/ renew --days 45 --renew-hook "nginx -s reload" 
```

Comment:
can we add `staging.cantusdatabase.org` to this line (which has now been set up - hooray!), and then test it on staging before we commit to running it on Production?

Reply:
For testing or to serve staging over https? In any case, I think it's a replace rather than add, since `cantusdatabase.org` won't validate on the staging server. But if we want to serve staging over https long-term, then we should maybe use an environment variable or something to switch between staging and production.

Reply:
With Ansible you could template this file and deploy it, with the appropriate host name, to the right server. I would recommend that approach.